### PR TITLE
Add test code to fix issue #7

### PR DESCRIPTION
**Pull request by kierenblack**
Adds a replacement tinylora library file and also test code to test the 32u4 board with the CO2 sensor

### CODE REVIEW

File: co2_sensor_32u4/32u4_co2.ino [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+void setup() {
+  Serial.begin(9600);  //display on serial monitor
+  while (!Serial)
+    ;
+
+  Serial1.begin(9600);
+
+  myMHZ19.begin(Serial1);    // Initialize the sensor with hardware serial port 1
+  
+  delay(20000000); // Warmup delay before calibration
```
Comment (SamanthaCra) [inline co2_sensor_32u4/32u4_co2.ino:59]: Was there a requirement for the delay? Could that be added as a comment just so we know not to remove it?

**Review by SamanthaCra — APPROVED** [review]
Looks good, thanks for diving into the libraries and fixing this.

File: co2_sensor_32u4/32u4_co2.ino (hunk 1 above)
Comment (kierenblack) [reply co2_sensor_32u4/32u4_co2.ino:59]: This is to allow the CO2 sensor to warm up before calibrating and collecting data

**Review by fuzzyers — APPROVED** [review]
My only concern would be the readability of this it might be better if we create a function for sending and a function for getting the  data which would allow us to tidy up loop and make it more readable

**Pull request merged**